packages/react-native-dogfood/src/components/LobbyViewComponent.tsx

```diff
@@ -0,0 +1,62 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { LobbyView, theme } from '@stream-io/video-react-native-sdk';
+import React from 'react';
+import { Pressable, StyleSheet, View, Text } from 'react-native';
+import { MeetingStackParamList } from '../../types';
+
+type LobbyViewComponentType = NativeStackScreenProps<
+  MeetingStackParamList,
+  'MeetingScreen' | 'GuestMeetingScreen'
+> & {
+  callId: string;
+};
+
+export const LobbyViewComponent = ({
+  callId,
+  navigation,
+  route,
+}: LobbyViewComponentType) => {
+  return (
+    <View style={[StyleSheet.absoluteFill, styles.container]}>
+      <LobbyView />
+      {route.name === 'MeetingScreen' ? (
+        <Pressable
+          style={styles.anonymousButton}
+          onPress={() => {
+            navigation.navigate('GuestModeScreen', { callId });
+          }}
+        >
+          <Text style={styles.anonymousButtonText}>
+            Join as Guest or Anonymously
+          </Text>
+        </Pressable>
```

Comment:
This is can be moved to a separate component and reused.. as the same style and pressable is used twice and only onPress and text changes...

simpler to maintain that way

Reply:
I mean a seperate component in the same file, just for reusing.. DRY principle